hack/make.ps1

```diff
@@ -303,7 +303,10 @@ Function Run-UnitTests() {
 # Start of main code.
 Try {
     Write-Host -ForegroundColor Cyan "INFO: make.ps1 starting at $(Get-Date)"
-    $root=$(pwd)
+
+    # Get to the root of the repo
+    $root = $(Split-Path $MyInvocation.MyCommand.Definition -Parent | Split-Path -Parent)
```

Comment:
Would it make sense to have a [`$SCRIPTDIR` variable](https://github.com/docker/docker/blob/master/hack/make.sh#L27), to keep the linux and windows variants a bit comparable?